rosetta/cosmos/launchpad/account_test.go

```diff
@@ -1,6 +1,25 @@
 package launchpad
 
-import "testing"
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+	"github.com/stretchr/testify/require"
+)
 
 func TestLaunchpad_AccountBalance(t *testing.T) {
+	adapter := NewLaunchpad("http://localhost:1317/")
+
+	res, err := adapter.AccountBalance(context.Background(), &types.AccountBalanceRequest{
+		NetworkIdentifier: nil,
+		AccountIdentifier: &types.AccountIdentifier{
+			Address: "cosmos15f92rjkapauptyw6lt94rlwq4dcg99nncwc8na",
+		},
+		BlockIdentifier: nil,
+	})
+	require.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Reply:
Yeah, the problem is that is not a real error but that types.Error defined from rosetta. But I tried! :D

Reply:
ah, sorry. I haven't noticed that.